server/src/main/java/org/elasticsearch/script/StoredScriptSource.java

```diff
@@ -329,28 +307,9 @@ public static StoredScriptSource parse(BytesReference content, XContentType xCon
                 } else {
                     throw new ParsingException(parser.getTokenLocation(), "unexpected token [" + token + "], expected [{, <source>]");
                 }
-            } else if (TEMPLATE_PARSE_FIELD.getPreferredName().equals(name)) {
-
-                DEPRECATION_LOGGER.deprecated("the template context is now deprecated. Specify templates in a \"script\" element.");
-
-                token = parser.nextToken();
-                if (token == Token.VALUE_STRING) {
-                    String source = parser.text();
-
-                    if (source == null || source.isEmpty()) {
-                        DEPRECATION_LOGGER.deprecated("empty templates should no longer be used");
-                    }
-
-                    return new StoredScriptSource(Script.DEFAULT_TEMPLATE_LANG, source, Collections.emptyMap());
-                } else {
-                    return parseRemaining(token, parser);
-                }
-            } else if (TEMPLATE_NO_WRAPPER_PARSE_FIELD.getPreferredName().equals(name)) {
```

Comment:
`TEMPLATE_NO_WRAPPER_PARSE_FIELD` and `TEMPLATE_PARSE_FIELD` constants can be removed now right?

Reply:
Yes, I already did this. I will commit this together with the fix for the failing test.